tests/ui/inference/need_type_info/concrete-impl.rs

```diff
@@ -12,5 +12,7 @@ impl Ambiguous<Two> for Struct {}
 fn main() {
     <Struct as Ambiguous<_>>::method();
     //~^ ERROR type annotations needed
+    //~| cannot infer type of the type parameter `A`
     //~| ERROR type annotations needed
+    //~| cannot infer type of the type parameter `A`
```

Comment:
I've added the label as an error annotation to safeguard against future accidental overwrites via `--bless`. Let me know if you deem this unnecessary.

Reply:
Seems fine.